companion commit for rstudio/rstudio-pro#2030 (#7970)
rstudio/rstudio: NEWS.md

### RStudio Server Pro

> * **DEPRECATED:** The option `auth-proxy-require-hmac` is no longer operational and a warning will be issued to the logs if used. See the documentation on proxy authentication for alternatives. (Pro #2029)

Reviewer: Nit: let's not lead with this as the very first thing people read about what we've done in this release! (put it towards the end of the notes)

* Changed the command to retrieve Slurm resource utilization to be run as the current user rather than the `slurm-service-user` (Pro #1527)
* Reduced supurflous log messages in the Slurm Launcher Plugin log file about non-RStudio jobs in Slurm (Pro #1528)
* Allow administrators to disabile the ability to set resource reqeusts on jobs launched through the Slurm Launcher Plugin (Pro #1948)

> * **DEPRECATED:** The option `auth-proxy-require-hmac` is no longer operational and a warning will be issued to the logs if used. See the documentation on proxy authentication for alternatives. (Pro #2029)

Author: ```suggestion
* **REMOVED:** The option `auth-proxy-require-hmac` is no longer operational and RStudio will not run if present. See the documentation on [Proxied Authentication] for alternatives to secure RStudio. (Pro #2029)
```

> * **REMOVED:** The option `auth-proxy-require-hmac` is no longer operational and RStudio will not run if present. See the documentation on [Proxied Authentication] for alternatives to secure RStudio. (Pro #2029)

Author: ```suggestion
* **REMOVED:** The option `auth-proxy-require-hmac` is no longer operational and RStudio will not run if present. See the documentation on [Proxy Security Considerations] for alternatives to secure RStudio. (Pro #2029)
```

> * **REMOVED:** The option `auth-proxy-require-hmac` is no longer operational and RStudio will not run if present. See the documentation on [Proxy Security Considerations] for alternatives to secure RStudio. (Pro #2029)

Author: ```suggestion
* **RETIRED:** The option `auth-proxy-require-hmac` is has been retired and it is no longer operational. RStudio will not start if enabled. See the documentation on [Proxy Security Considerations] for alternatives to secure RStudio. (Pro #2029)
```
